docs/osquery/osquery.asciidoc

```diff
@@ -0,0 +1,394 @@
+.. To replay a query, click image:images/play-icon.png[Right-pointing triangle].
+.. To view the query <<osquery-results,results>> and <<osquery-status,status>>,
+click image:images/table-icon.png[Table icon].
++
+[role="screenshot"]
+image::images/live-query-check-results.png[Results of OSquery]
+
+
+[float]
+[[osquery-schedule-query]]
+== Schedule queries
+
+Group and schedule queries to run on a specified interval, in seconds.
+For example, you might create a group that checks
+for IT compliance-type issues and
+another group that monitors for evidence of malware.
+
+. Open the **Scheduled query groups** tab.
+
+. Click a group name to view its details.
++
+Details include the last time it ran, how many results were returned, and the number of agents it ran against.
+If there are errors, expand row to view the details.
++
+[role="screenshot"]
+image::images/scheduled-query-group.png[Shos last results last time it ran, how many results returned, number of agents it ran against,  Is
+it actually running and if there are errors]
+
+. Click *Edit* to make changes.
+.. To add a query to the group, click *Add query*, and then enter the query ID and interval.
+Optionally, set the minimum Osquery version and platform
+or <<osquery-map-fields,map ECS fields>>.
+.. To upload queries from a .conf query pack, drag the pack to the drop zone under the query table. To explore the community packs that Osquery publishes, click *Example packs*.
+. Click *Save query*. The queries run once the policy receives the update.
+. View scheduled history results in Discover or Lens.
+
+
+[float]
+[[osquery-map-fields]]
+== Map Osquery fields to ECS fields
+
+When scheduling queries, you can optionally map query results to fields in
+the {ecs-ref}/ecs-reference.html[Elastic Common Schema] (ECS),
+so that you can standardize your Osquery data for use across detections, machine learning,
+and any other areas that rely on ECS-compliant data.
+The query results then include both the original `osquery.<field>`
+and the mapped ECS field. For example, if you update a query to map `osquery.name` to `user.name`
+in ECS, the query results include both fields.
+
+. Edit a scheduled query group, and then click the edit icon for the query that you want to map.
+
+. In **ECS mapping**, select which Osquery result fields you want to map to ECS fields.
++
+The fields available in the **Osquery.results** column are based on the SQL query entered
+and only include fields that will be returned by the query.
+
+. When mapping fields:
+.. To add a new row for additional fields to map, click the plus icon.
+.. To remove any mapped rows, click the trash icon.
+
+. Click *Save* to save changes to the query.
+
+. Click *Save query* to save changes to the group.
+
+
+[float]
+[[osquery-manage-query]]
+== Edit your saved queries
+
+Add or edit saved queries on the *Saved queries* tab.
+
+1. Go to saved queries
+2. From the saved queries, click **Add saved query**. or edit icon
```

Comment:
I think there's an extra period in the middle of the sentence - it should probably just be:

> From the saved queries, click **Add saved query** or the edit icon.